partner_noncommercial/readme/USAGE.rst

```diff
@@ -1,3 +1,3 @@
-This module creates a main menu All partners. There it gives special
+This module creates a main menu Partners. There it gives special
```

Comment:
This is technically not true any more, we should maybe change this to:

"This module adds specific filters, groupings and modifies some naming to make the existing main menu Contacts easier to use for non-commercial entities".

Reply:
I adapted the usage.